x-pack/plugins/apm/public/components/app/error_group_details/index.tsx

```diff
@@ -73,10 +56,10 @@ function getShortGroupId(errorGroupId?: string) {
 
 function ErrorGroupHeader({
   groupId,
-  isUnhandled,
+  occurrencesCount,
 }: {
   groupId: string;
-  isUnhandled?: boolean;
+  occurrencesCount?: number;
```

Comment:
what happens if `ocurrencesCount` is undefined?

Reply:
`ErrorGroupHeader` has two renders, one when we don't have errors data and one when we have, if we have errors data error occurrences would be defined
```
if (!errorDistributionData || !errorSamplesData) {
    return <ErrorGroupHeader groupId={groupId} />;
  }
```
```
  <ErrorGroupHeader
        groupId={groupId}
        occurrencesCount={errorSamplesData.occurrencesCount}
      /> 
```